plugins/pulp_rpm/plugins/db/models.py

```diff
@@ -236,55 +461,114 @@ def rpm_search_dicts(self):
 
 
 class PackageGroup(Package):
-    UNIT_KEY_NAMES = ('id', 'repo_id')
-    TYPE = ids.TYPE_ID_PKG_GROUP
-
-    def __init__(self, id, repo_id, metadata):
-        Package.__init__(self, locals())
-        # these attributes should default to False based on yum.comps.Group.parse
-        for name in ('default', 'user_visible'):
-            if self.metadata.get(name) is None:
-                self.metadata[name] = False
+    # TODO add docstring to this class
+    package_group_id = mongoengine.StringField(required=True)
+    repo_id = mongoengine.StringField(required=True)
```

Comment:
The `repo_id` caught my eye here.  Can I assume this is YUM metadata and has nothing to do with pulp repositories?